doc/admin-guide/configuration/index.en.rst

```diff
@@ -33,3 +33,4 @@ Proxy Cache Configuration
    transparent-forward-proxying.en
    hierarchical-caching.en
    proxy-protocol.en
+   hrw4u.en
```

Comment:
This seems out of place here since this describes the DSL for the header rewrite plugin. The other stuff here concerns core proxy configuration.

Honestly, I suggest we simply replace the current header rewrite docs with what you have above and link old docs from there in case people want to reference the old syntax. (In the transition period, people will need docs for the old language to help understand their scripts.)